tests/client/test_items.py

```diff
@@ -36,3 +39,45 @@ def test_items_list(spider, json_and_msgpack):
     assert o[0] == {'id': 0, 'data': 'data0'}
     assert o[1] == {'id': 1, 'data': 'data1'}
     assert o[2] == {'id': 2, 'data': 'data2'}
+
+
+def test_items_list_iter(spider, json_and_msgpack):
+    job = spider.jobs.run(meta={'state': 'running'})
+    job = normalize_job_for_tests(job)
+    _add_test_items(job)
+    job.finish()
+
+    o = job.items.list_iter(chunksize=2)
+    assert next(o) == [
+        {'id': 0, 'data': 'data0'},
+        {'id': 1, 'data': 'data1'},
+    ]
+    assert next(o) == [
+        {'id': 2, 'data': 'data2'},
+    ]
+    with pytest.raises(StopIteration):
+        next(o)
+
+
+def test_items_list_iter_with_start_and_count(spider, json_and_msgpack):
+    job = spider.jobs.run(meta={'state': 'running'})
+    job = normalize_job_for_tests(job)
+    _add_test_items(job, size=10)
+    job.finish()
+
+    o = job.items.list_iter(chunksize=3, start=3, size=7)
```

Comment:
I noticed one line is not executed by tests https://codecov.io/gh/scrapinghub/python-scrapinghub/compare/92ff0968871c95dd00b078bcf0f6f5eb944c0f2c...c4ef768d3dda78a6de3fa119db0652ec891e84fa/diff

```suggestion
    o = job.items.list_iter(chunksize=3, start=3, count=7)
```

Reply:
Thanks for spotting @manycoding ! Much appreciated! just fixed it now and all CI tests are passing. 🎉